sds_data_manager/lambda_code/SDSCode/create_schema.py

```diff
@@ -0,0 +1,95 @@
+import json
+import os
+
+import boto3
+import psycopg2
+import requests
+from psycopg2 import Error
+
+# TODO: The code in this lambda will be updated in the next PR
+# to reflect the schema that will be needed by the SDC.
+# Its current state is the bare minimum to get a table
+# created to allow files to be indexed.
+
+# Add code to get Secret instead
```

Comment:
Could you expand on this a bit? Why is S3 in particular needed for this? And what is the alternative method we are avoiding by doing it this way?

Reply:
That comment and line of code are confusing because they're completely unrelated and neither are needed lol Good catch, I'll delete these two lines:

```python
# Add code to get Secret instead
s3 = boto3.client("s3")
```